test/functional/feature_rbf.py

```diff
@@ -594,7 +594,7 @@ def test_incremental_relay_feerates(self):
         node = self.nodes[0]
         for incremental_setting in (0, 5, 10, 50, 100, 234, 1000, 5000, 21000):
             incremental_setting_decimal = incremental_setting / Decimal(COIN)
-            self.log.info(f"-> Test -incrementalrelayfee={incremental_setting_decimal:.8f}sat/kvB...")
+            self.log.info(f"-> Test -incrementalrelayfee={incremental_setting:.8f}sat/kvB...")
```

Comment:
In 636fa219d37f86067d996c86fada286cedc0d78e "test fixups"

Can drop the `.8f`, otherwise these are printed with 8 decimal places of 0's that are not necessary.

Reply:
`-> Test -incrementalrelayfee=100.00000000sat/kvB...` doesn't seem very useful indeed. 

mining_basic reports `-> Test -blockmintxfee=0.00000500 (500 sat/kvB)...` which seems strictly better, as it gives a value that can actually be used with the config param as well as a sats figure?